vunit/vivado/vivado.py

```diff
@@ -119,12 +133,22 @@ def run_vivado(tcl_file_name, tcl_args=None, cwd=None, vivado_path=None):
     """
     Run tcl script in Vivado in batch mode.
 
-    Note: the shell=True is important in windows where Vivado is just a bat file.
+    :param tcl_file_name: Path to tcl file
+    :param tcl_args: tcl arguments passed to Vivado via the ``-tclargs`` switch
+    :param cwd: Passed as ``cwd`` to :func:`subprocess.check_call`
+    :param vivado_path: Path to Vivado install directory. If ``None``, the
+        environment variable ``VUNIT_VIVADO_PATH`` is used if set. Otherwise,
+        rely on that ``vivado`` is in the path.
     """
-    vivado = "vivado" if vivado_path is None else str(Path(vivado_path).resolve() / "bin" / "vivado")
-    cmd = f"{vivado} -nojournal -nolog -notrace -mode batch -source {str(Path(tcl_file_name).resolve())}"
+    vivado = (
+        str(Path(vivado_path).resolve() / "bin" / "vivado")
+        if vivado_path is not None
+        else environ.get("VUNIT_VIVADO_PATH", "vivado")
+    )
```

Comment:
I think it is easier to comment line by line.

This line introduces the environment variable (documented above).

Reply:
Fixes #811